docs/changes.rst

```diff
@@ -2,6 +2,11 @@
 Change Log
 ==========
 
+?X.Y.Z
```

Comment:
😝 BTW, I've been using `?.?.?` with one of those slots as an `X` to indicate the degree of change associated with log message. The hope was that it would make it easier to comprehend come tagging time. For example, `?.?.X` for bug fix.